tests/models/test_modeling_common.py

```diff
@@ -1877,58 +1859,86 @@ def test_hotswapping_compiled_model_linear(self, rank0, rank1):
     @parameterized.expand([(11, 11), (7, 13), (13, 7)])  # important to test small to large and vice versa
     def test_hotswapping_compiled_model_conv2d(self, rank0, rank1):
         # It's important to add this context to raise an error on recompilation
+        if "unet" not in self.model_class.__name__.lower():
```

Comment:
The comment above this line should move below this line, as it relates to the context manager.